docs/content/rust-bindings.zh.md

```
@@ -0,0 +1,90 @@
+# Rust 绑定
+
+ccap 提供了完善的 Rust bindings，对应 crates.io 上的 `ccap-rs` crate。
+
+- Crate：https://crates.io/crates/ccap-rs
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)